tests/L0/run_fused_layer_norm/test_fused_layer_norm.py

```diff
@@ -99,6 +187,16 @@ def _prep_layers(normalized_shape, elementwise_affine, dtype):
     return native, fused
 
 
+def _prep_rms_layers(normalized_shape, elementwise_affine, dtype):
+    native = apex.normalization.FusedRMSNorm(
+        normalized_shape=normalized_shape, elementwise_affine=elementwise_affine
+    )
+    fused = apex.normalization.FusedRMSNorm(
+        normalized_shape=normalized_shape, elementwise_affine=elementwise_affine
+    ).cuda()
+    return native, fused
```

Comment:
this testing won't do much good as it's comparing to itself :)

Since there isn't `torch.nn.RMSNorm`, perhaps writing one out in plain python?

Reply:
It's a bit opaque here, but the "native" version is computed on CPU which dispatches to a manual plain python version sourced from `T5LayerNorm`: https://github.com/NVIDIA/apex/blob/028ef04cb7a5ca79a5c2929fb4416209c27555ac/apex/normalization/fused_layer_norm.py#L410

https://github.com/NVIDIA/apex/blob/028ef04cb7a5ca79a5c2929fb4416209c27555ac/apex/normalization/fused_layer_norm.py#L16

Reply:
Thank you for explaining this nuance, @eqy. I can see it now.